app/views/facilities/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%=form_for @facility do |f|%>
+    <div class="p-2">
+        <%=f.text_field  :name ,placeholder:"Name of Facility"%>
```

Comment:
[Style]
The spacing of this line (and the other lines in this form) is a bit off. There should always be spaces between your code and the ERB tags.

```suggestion
        <%= f.text_field :name, placeholder: "Name of Facility" %>
```